blockchain/indexers/cfindex.go

```diff
@@ -255,6 +255,37 @@ func (idx *CfIndex) DisconnectBlock(dbTx database.Tx, block *btcutil.Block,
 	return nil
 }
 
+// PruneBlock is invoked when an older block is deleted after it's been
+// processed.  For cfindx, PruneBlock is exactly the same as disconnecting a block.
+// TODO (kcalvinalvin): Consider keeping the filters at a later date to help with
```

Comment:
I guess we could also support fast rescan using compact filters in the future when pruning? Maybe it would be a great idea to have an option like `prune-cfilters=true/false` that is `false` by default (since filters for mainnet are currently only around 8 GB).

Reply:
I just realized that the fast rescan for bitcoin core is for the wallet. Since btcd doesn't have any wallet functionality with itself, there isn't a need for the compact filters for this reason.

Does btcwallet ask for the filters? Might be worth keeping the filters around for it?

Reply:
> Does btcwallet ask for the filters(not familiar with the codebase)? Might be worth keeping the filters around for it?

We don't do that _yet_ but it's something we've been meaning on adding.